### PR TITLE
Fix: Fixed autogenerated description

### PR DESCRIPTION
**Pull request by ctasada**
When defining an `@AsyncOperation` description like in

```
@AsyncOperation(
    channelName = "channel-name",
    description = "My channel description")
```

The generated AsyncAPI schema was always showing `Auto-generated description`
now properly shows the defined `description` value.

### CODE REVIEW

**Review by sam0r040 — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution, I just made a suggestion on how to resolve the FIXME comment.

**Review by sam0r040 — CHANGES_REQUESTED** [review]
Thanks a lot for your contribution, I just made a suggestion on how to resolve the FIXME comment.

File: springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/operationdata/AbstractOperationDataScanner.java [hunk 1]
```diff
@@ -96,6 +101,7 @@ private Message buildMessage(OperationData operationData) {
         return Message.builder()
                 .name(payloadType.getName())
                 .title(modelName)
+                // FIXME? Using operation description in the message?
                 .description(operationData.getDescription())
```
[the remark this answers is not in the document]
Comment (ctasada) [reply springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/operationdata/AbstractOperationDataScanner.java:105]: @sam0r040 I agree that we need to change it. If I remove the line, the message description becomes `null`

I think the proper solution here is that we need to obtain the value from the `@Schema(description = "Message/Schema description goes here")`

I can remove the line and "fix" all the tests (validating the description is `null`) or leave it like it's now (the bug was already there). Next week I can work in another PR to address this issue.

What do you prefer?

[the remark this answers is not in the document]
Comment (timonback) [reply springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/operationdata/AbstractOperationDataScanner.java:105]: As a first step, I am also fine with removing the line and letting the field be implicitly `null`, which isn't rendered into the final document.

I like the idea to use the Schema annotation as a source. Maybe we can utilize it through the SchemaService?

But probably better to fix the description first, and put the feature of the message description in a new PR.

**Comment by timonback** [comment]
@ctasada Is there anything anything else you like to see being included in the next release?

File: springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/operationdata/AbstractOperationDataScanner.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (ctasada) [reply springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/operationdata/AbstractOperationDataScanner.java:105]: @sam0r040 @timonback I will update the PR removing the line then.

**Comment by ctasada** [comment]
> @ctasada Is there anything anything else you like to see being included in the next release?

Nothing urgent. If we can get another release done in 2-4 weeks I think I'm good 😉

**Review by timonback — APPROVED** [review]

**Pull request merged**